app/main.ipynb

```diff
@@ -268,18 +272,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "sum_by_y_axis = np.sum(A, axis=0)",
```

Comment:
This line seems to be a duplicate from Task 5.3. For Task 5.4, you need to implement the multi-dimensional selection to obtain the specified sub-array.